### ISSUE TITLE
loss.backward is too slow

### ISSUE THREAD

**Issue by fengchuanpeng** [JCruan519/VM-UNet#50]
Hello，that is mainly because there is something wrong which lead the loss.backward so slow， the batch_size is  five，one  iter  need 725s

**Comment by tmax-cn** [comment]
same issue here, any updates?

**Comment by JCruan519** [comment]
@tmax-cn @fengchuanpeng 
Hello, when setting the batch size to 32, training one epoch on an A6000GPU takes 100s. I suggest first checking for any environmental issues (ensure that mamba-related libraries are correctly installed), and then verify whether the GPU is being fully utilized during training.
